susApp/www/js/DrawerLinks.js

```diff
@@ -0,0 +1,51 @@
+angular.module('DrawerLinks', [])
+
+.controller('DrawerController', function($state, $scope, $location, $ionicLoading, $window, $ionicHistory, $timeout, $localStorage) {
+
+
+  $scope.username = localStorage.getItem('user_username');
```

Comment:
We should consider moving these kind of values into constant so that it is easier to find

```
.constant('USER', {
  username: 'user_username'
})
```
Then pass "USER" as a param into the controller and use like;

`localStorageService.getItem(USER.user_username);`